recipes/scitime/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: scitime
+  version: {{ version }}
+
+source:
+  url: https://github.com/nathan-toubiana/scitime/archive/{{ version }}.tar.gz
+  sha256: 05f11512343bbaf521c6358e8321a8555d0ef4fe41a76ced9a8234d8b8bbab5b
+
+build:
+  number: 1
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools >=3.3
+    - pip
+    - scikit-learn
+    - psutil
+    - joblib
+    - pandas
+  run:
+    - python
```

Comment:
The host and run requirements look backwards here. Host should only be stuff needed to install. Run is actual runtime dependencies.